checkers/model_form.py

```diff
@@ -0,0 +1,55 @@
+import ast
+
+from .checker import Checker
+from .issue import Issue
+
+
+class DJ06(Issue):
+    code = 'DJ06'
+    description = 'ModelForm.Meta should not set exclude, set fields instead'
+
+
+class ModelFormChecker(Checker):
+
+    def checker_applies(self, node):
+        for base in node.bases:
+            if self.is_model_form_attribute(base) or self.is_model_form_name(base):
+                return True
+        return False
+
+    def is_model_form_name(self, base):
+        return (
+            isinstance(base, ast.Name) and
+            base.id == 'ModelForm'
+        )
+
+    def is_model_form_attribute(self, base):
+        return (
+            isinstance(base, ast.Attribute) and
+            isinstance(base.value, ast.Name) and
+            base.value.id == 'models' and base.attr == 'ModelForm'
+        )
+
+    def run(self, node):
+        """
+        Captures the use of exclude in ModelForm Meta
+        """
+        if not(self.checker_applies(node)):
```

Comment:
Does it feels more pythonic in writing this way?

```python
if not self.checker_applies(node):
```

We don't need to cast this bool expression using `()`.

Reply:
👍 fixed